pkg/auth/iam/iam.go

```diff
@@ -477,3 +500,44 @@ func respondError(httpStatus, errorCode int, errorMessage string) restful.Servic
 		Message: string(messageByte),
 	}
 }
+
+// SetDevStackTracer used for activate verbose error message in non-prod environment
+func SetDevStackTracer() {
+	DevStackTraceable = true
+	realmName, realmNameExists := os.LookupEnv("REALM_NAME")
+	if !realmNameExists {
+		DevStackTraceable = false
+		return
+	}
+
+	realmLive, realmLiveExists := os.LookupEnv("REALM_LIVE")
+	if !realmLiveExists {
+		return
```

Comment:
can we have a default value for the realm_value if it's not exists, otherwise it will default true from you assignment, but it's a security issue to expose more dev stack

Reply:
sorry @angelo-chan, I think I miss understand your comment "_the realmLives should have default values_" above :bowing:

We still need default value using constant like `DefaultRealmLive= "prod,live,demo,trial"`